rewards/README.md

```diff
@@ -4,40 +4,86 @@ cover: ../.gitbook/assets/cover/rewards_banner.png
 
-## Data Farming
+### Active Rewards
 
-![DF Rewards Page](../.gitbook/assets/rewards/df_rewards_page.png)
+Active Rewards is the second reward streamm within Data Farming.  
 
-[Data Farming 101](df-intro.md) introduces the different reward systems, how they work, and how to access them. By the end of the page, you should be more familiar with how Data Farming works and able to take next steps to curate assets.
+It's an evolving stream that focuses on driving fundamental traction and growth of the protocol.  
 
-[Data Farming Background](df-max-out-yield.md) will provide you with more intuitions about Data Farming, briefly explain the Reward Function, and how the program evolved over time.
+For participants that are more engaged and want to increase their yield, the [Active Rewards](df-intro.md#what-are-active-rewards) stream is broken down into multiple substreams.  
 
-## Delegation
+This offers contributors a variety of ways to help Ocean Protocol grow and increase your Data Farming rewards:
+1. Volume DF
+1. Challenge DF
 
-[Delegation](../user-guides/how-to-data-farm.md#how-to-delegate-your-active-rewards) will teach you how to share your veOCEAN allocation power to other users who can manage Data Farming for you.
+#### Curate Data with Volume DF
 
-Once delegated, rewards will be sent to the wallet address you delegated to. The delegation receiver is in charge of your rewards and the process of returning those back to you.
+If you have locked your OCEAN tokens in exchange for veOCEAN, this is the ideal opportunity for you to utilize those veOCEAN tokens to earn higher yield.
+
+You can learn about [the design of Volume DF](df-volumedf.md) to learn about it's substreams. Here, curators earn rewards for curating datasets that are driving sales and helping Ocean Protocol achieve traction.
+
+Once you're familiar with the concepts there, you can then read the following user guide on [how to participate in Volume DF](../user-guides/data-farming/how-to-volumedf.md).
+
+## Delegate Curation
+
+The [Delegation](df-volumedf.md#delegating-voting-power) docs will teach you about the concepts of delegating your veOCEAN allocation power to another wallet that will manage Volume DF for you.
+
+You can then use the [how to delegate](../user-guides/data-farming/how-to-delegate.md) user guide to follow the steps needed to successfully delegate.
+
+Please note, once delegated, rewards will be sent to the wallet address you delegated to. The delegation receiver is in charge of managing your Allocation Power and cut from the rewards. 
+
+#### Predict with Challenge DF
+
+In this substream, Data Scientists can win competitions by participating in [Challenge DF](df-challengedf.md). You will obtain overview of what it is, how it works, the rules, and other details.
+
+Challenge DF requires participants to submit algorithms that predict the price of ETH correctly in order to win rewards.
```

Comment:
People need to submit predictions not algorithms

Reply:
updated